documentation/content/developmentworkflow.rst

```diff
@@ -0,0 +1,110 @@
+========================
+  Development workflow
+========================
+
+.. contents::
+   :depth: 1
+   :local:
+   :backlinks: none
+
+.. highlight:: console
+
+kw is developed using Git and is hosted at Github. If you want to understand
+our development workflow better, clone our repository to have your local copy::
+
+  git clone https://github.com/kworkflow/kworkflow.git
+
+Install
+-------
+Installing kw is one of the first steps to understanding its development
+workflow. After cloning the repository, read the page related to
+:ref:`installing and removing<install-and-remove-kw>`.
+
+Tests
+-----
+To make sure our code is working as it is supposed to, we add
+:ref:`tests<tests>` to check that.
+
+All our tests can be found in the `tests` folder. Notice that every test file
+name ends with test before the .sh extension (i.e. `mk_test.sh` and
+`statistics_test.sh`). This is done so that the `run_tests.sh` file can detect
+those tests and run or list them.
+
+Unit tests
+==========
+Unit testing is an essential part of our development workflow. Ideally, every
+new function should have a test related to it. So, if you're working on a new
+feature, added a new function, or changed the behavior of an existing one, it's
+fundamental to make the necessary ajustments related to testing.
+
+To see how to run individual test files, check our :ref:`page on that<tests>`.
+
+Run kw without installing it
```

Comment:
It may be a good idea to explicit the development benefits of this, I think the way it is written may lead a new user to believe this is another way of using kw while it is primarily focused on making development more agile